looper/__init__.py

```diff
@@ -39,7 +40,14 @@
 # the locations of some of the peppy declarations. Effectively, concentrate
 # the connection between peppy and looper here, to the extent possible.
 
-__all__ = ["Project", "PipelineInterface", "SubmissionConductor"]
+__classes__ = ["ComputingConfiguration"]
+__functions__ = ["select_divvy_config"]
+__all__ = (
+    ["Project", "PipelineInterface", "SubmissionConductor"]
```

Comment:
do you want to move Project, PipelineInterface, and SubmissionConductor into "classes" so this is consistent? Or just put everything in all, but pick one way?

Reply:
Yeah, we can do it one way, I'll change that to be more consistent